docs/developers/filter_tutorial.md

```diff
@@ -14,12 +14,12 @@ To make sure you are working in the correct environment, with exact version of d
 
 ```bash
 # 2.3GB compressed
-docker pull julienpeloton/fink-ci:latest
+docker pull julienpeloton/fink-ci-ztf:latest
 
 # Assuming you are in /path/to/fink-filters
 docker run -t -i --rm -v \
-  $PWD:/home/libs/fink-filters \ # (1)!
-  julienpeloton/fink-ci:latest bash
+  $PWD:/home/libs/fink-filters \
```

Comment:
Actually the `# (1)!` is here on-purpose (it generates a (+) sign on the website with extra information)

Reply:
ok I put it back!